pkg/cmd/init.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/openshift/microshift/pkg/util/cryptomaterial"
 )
 
+var DataDir = config.FindDataDir()
```

Comment:
I think this should be written in a way that guarantees everything in the process uses the same directory. By exporting FindDataDir, which can return different values from call to call, there's the risk of future issues with multiple data directories.

Reply:
I can construct a struct with directory-related configuration that will be generated at the beginning and pass it along as a parameter to those functions.

Reply:
If we want to move to a single, fixed location for the data dir (`/var/lib/microshift`), then it means this simply reduces to an accessor function to a global-scope constant, or? What I don't understand is why we want to do this if we wire through the instance config (`cfg`) anyway and often need that info in the same functions?